annotations-runtime/src/test/java/com/github/dynamicextensionsalfresco/webscripts/ExceptionHandlerTest.java

```diff
@@ -10,22 +14,40 @@
 @DirtiesContext(classMode = ClassMode.AFTER_EACH_TEST_METHOD)
 public class ExceptionHandlerTest extends AbstractWebScriptAnnotationsTest {
 
-	@Autowired
-	private ExceptionHandlerExample handler;
-
-	@Test
-	public void testHandleExceptionOfOneType() {
-		handleGet("/throwIllegalArgumentException");
-		assertNotNull(handler.illegalArgumentException);
-		assertNotNull(handler.throwable);
-		assertNull(handler.illegalStateException);
-	}
-
-	@Test
-	public void testHandleExceptionOfAnotherType() {
-		handleGet("/throwIllegalStateException");
-		assertNull(handler.illegalArgumentException);
-		assertNotNull(handler.throwable);
-		assertNotNull(handler.illegalStateException);
-	}
+    @Autowired
+    private ExceptionHandlerExample handler;
+
+    @Test
+    public void testHandleExceptionOfOneType() {
+        handleGet("/throwIllegalArgumentException");
+        assertNotNull(handler.illegalArgumentException);
+        assertNotNull(handler.throwable);
+        assertNull(handler.illegalStateException);
+    }
+
+    @Test
+    public void testHandleExceptionOfAnotherType() {
+        handleGet("/throwIllegalStateException");
+        assertNull(handler.illegalArgumentException);
+        assertNotNull(handler.throwable);
+        assertNotNull(handler.illegalStateException);
+    }
+
+    @Test
+    @Ignore // @ExceptionHandler annotated default method in interfaces do not work for Alfresco <= 5.
```

Comment:
Instead of `@Ignore`-ing this test, you can use `assumeTrue("Annotated default methods in interfaces is only supported starting from Alfresco 6", <check Alfresco >=6>)`

Reply:
:+1: